packages/kbn-ai-playground/utils/api.ts

```diff
@@ -0,0 +1,150 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
```

Comment:
nitpick - can we move the the server only parts into a server/utils here? just so its easier to navigate.

Reply:
Of course